draft-ar-pquip-pqc-engineers.md

```diff
@@ -370,11 +370,9 @@ where pk is public key, sk is secret key, ct is the ciphertext representing an e
 
 ~~~~~
 
-### Interactivity in PQC KEM and Diffie-Hellman (DH) Key Exchange
+### Authenticated Key Exchange (AKE)
 
-PQ KEMs are interactive in nature because it involves back-and-forth communication to negotiate and establish the shared secret key and is unlike the Diffie-Hellman (DH) Key exchange (KEX) or RSA Key Transport which provide non-interactive key exchange (NIKE) property. NIKE is a cryptographic primitive which enables two parties, who know each others public keys, to agree on a symmetric shared key without requiring any real-time interaction.  Think of encrypted email where the content needs to be encrypted and sent even if the receiving device containing the decryption keys (e.g., phone or laptop) is currently offline.  The following figure illustrates a sample flow of (ephemeral-ephemeral) DH:
-
-EDNOTE (Mike Ounsworth): I actually disagree with this section. *Authenticated* Key Exchange with KEMs -- ie where both parties contribute a KEM public key to the overall session key -- is necessarily interactive in the way described in this section (RIP Signal Protocol), but single-sided KEM, for example where one person has a KEM key in a certificate and the other person simply wants to encrypt for it, as in S/MIME or OpenPGP email, is perfectly doable with non-interactive HPKE (RFC9180), which is actually explaied in the next section. I suggest this section be re-worked: a) re-titled to "Authenticated Key Exchange", and b) explain why we love this property of DH, and why we're all it tears about losing it, I have added some text below the diagram to attempt to do this, but this needs another editing pass.
+*Authenticated* Key Exchange with KEMs -- i.e., where both parties contribute a KEM public key to the overall session key -- is interactive in the way described in this section (RIP Signal Protocol), but single-sided KEM, for example where one person has a KEM key in a certificate and the other person simply wants to encrypt for it, as in S/MIME or OpenPGP email, is perfectly doable with non-interactive HPKE (RFC9180), which is explained in the next section. The following figure illustrates the Diffie-Hellman (DH) Key exchange:
```

Comment:
I am not familiar with RIP signal protocol. can you pick another example for Authenticated Key Exchange with KEMs like https://datatracker.ietf.org/doc/draft-ietf-lake-edhoc/. I suggest replacing "person" with 'peer" in the above paragraph.

Reply:
Okay, I will make the changes.

Reply:
Done.